app/src/main/java/com/group80/uoftinder/RecommendationPresenter.java

```diff
@@ -0,0 +1,27 @@
+package com.group80.uoftinder;
+
+import java.util.List;
+
+public class RecommendationPresenter implements RecPresenterInterface{
+    private GenerateCompatibilityList genCompatibilityList;
+
+    public RecommendationPresenter() {
+        genCompatibilityList = new GenerateCompatibilityList(this);
```

Comment:
I know we talked about this briefly in person, just wondering if you ever got the chance to double check that this is actually a valid way of calling? We might need to think this through more regarding interactions between the Interactor, Presenter, Presenter Interface, View, and View Interface

Reply:
I will test this out when I have a hold of everyone else's code and let you know. In the meantime, I will also do some googling to see if there's a better way of doing this and if what I did here is actually valid.